WaltonSustainabilityCentre-Requirements.txt

```diff
@@ -48,9 +48,9 @@ Originator: Karen Black, Centre Administrator
 Comments: GS
 
 REQ-ID: 010
-Description: The product shall identify additional equipment requirements
+Description: The product shall predict stock levels of regularly used resources, and notify staff when re-stocking is required.
```

Comment:
I'm not keen on the wording of this one. I know we don't have fit criteria yet but:

1. This suggests two separate atomic requirements
    1. Predicting stock levels
    1. Providing notifications
1. The interpretation of "regularly used" is ambiguous and could vary greatly.